pset-iterators.js

```diff
@@ -211,7 +272,21 @@ console.log('Problem 8:')
 
 // Add your code below this line
 
+const stepOne = presidents.map((firstName)=>{
+  return firstName.president.split(" ")[0]
+})
+
+
+
+console.log(stepOne);
+
+// const firstNameCount = stepOne.reduce((nameToAmount)=>{
+//
```

Comment:
You are heading in the right direction here! One possible solution is to create an empty object and fill it using the reduce method. [Here](https://github.com/jsd20200127/hw-03-iterators-solution/blob/c31d6d65d8f18a945cb6e59470aca10687d61aa3/pset-iterators.js#L251-L259) is an example of that.